lib/common/formats.js

```diff
@@ -34,29 +34,106 @@ function cssHeader() {
 }
 
 module.exports = {
-  'css/variables': function(dictionary) {
-    var to_ret;
-    return cssHeader() + ':root {\n' + _.map(dictionary.allProperties, function (prop) {
-      if (prop.attributes.category !== 'asset') {
-        to_ret = '  --' + prop.name + ': ' + prop.value + ';';
+  'css/fonts': function(dictionary) {
```

Comment:
This method is really long, I think functionality like this is better suited for a template. We can achieve the same functionality with this template:

```
<% _.forIn(properties.asset.font, function(font) { %>
  @font-face {
    font-family: <%= font.name.value %>;<% if (font.eot) { %>
    src: url('../<%= font.eot.value %>');
    src: url('../<%= font.eot.value %>?#iefix') format('embedded-opentype'),<% } if(font.ttf) { %>
      url('../<%= font.ttf.value %>') format('truetype'),<% } if(font.otf) { %>
      url('../<%= font.otf.value %>') format('otf'),<% } if(font.woff) { %>
      url('../<%= font.woff.value %>') format('woff'),<% } if(font.woff2) { %>
      url('../<%= font.woff2.value %>') format('woff2');<% } %>
  }
<% }); %>
```

Or maybe use this line `_.forIn(properties.asset.font, function(font) {` instead of looping over properties and creating these objects on the fly, and then follow the same type of logic in the template above to construct the string.